app/src/main/contrib/components/Responsibilities/Guidance/content/ResponsibilityGuidanceModelOutputsContent2021.tsx

```diff
@@ -0,0 +1,103 @@
+import * as React from "react";
+
+import {PageArticle} from "../../../../../shared/components/PageWithHeader/PageArticle";
+import {settings} from "../../../../../shared/Settings";
+import {mapStateToGuidanceContentProps, ResponsibilityGuidanceContentProps} from "./ResponsibilityGuidanceContentProps";
+import {connect} from "react-redux";
+
+
+export class ResponsibilityGuidanceModelOutputsContent2021Component extends React.Component<ResponsibilityGuidanceContentProps> {
+
+    render(): JSX.Element {
+        const centralEstimatesColumns = require("./2018-central-estimates-columns.png");
+
+        return <PageArticle title="Guidance on model outputs: how to generate and upload central estimates">
+
+            <div className="alert alert-primary">Guidance for
+                touchstone {this.props.touchstoneVersion.description}</div>
+
+            <div className="largeSectionTitle">Overview</div>
+            <p>The 2021 practice runs are not a requirement, but we encourage modellers to provide them to ensure their
+                model will work on Montagu for the full 2021 runs. For the practice runs, we expect only central
+                estimates, not stochastic estimates.</p>
+            <p>(For the full model runs, we will require both central and stochastic estimates.)</p>
+
+            <div className="largeSectionTitle">Countries</div>
+            <ul>
+                <li>JE groups: China, Pakistan, India</li>
+                <li>Men A & YF groups: Ethiopia, Nigeria</li>
```

Comment:
Pedantic point, and inherited from the original doc, but I think this would be more consistent with following line
```suggestion
                <li>Men A and YF groups: Ethiopia, Nigeria</li>
```